src/com/intellij/idea/plugin/hybris/project/compile/ProjectBeforeCompilerTask.kt

```diff
@@ -120,7 +120,7 @@ class ProjectBeforeCompilerTask : CompileTask {
             .withParameters(
                 "-Dfile.encoding=UTF-8",
                 "-cp",
-                classpath.joinToString(":"),
+                classpath.joinToString(";"),
```

Comment:
for Unix it is `:`, please make it OS specific

Reply:
Adjusted